src/include/rats-tls/cert.h

```diff
@@ -24,34 +26,16 @@ typedef struct {
 	uint32_t ias_report_signature_len;
 } attestation_verification_report_t;
 
-typedef struct {
-	uint8_t quote[8192];
-	uint32_t quote_len;
-} ecdsa_attestation_evidence_t;
-
-typedef struct {
-	uint8_t report[8192];
-	uint32_t report_len;
-} la_attestation_evidence_t;
-
-typedef struct {
-	uint8_t quote[8192];
-	uint32_t quote_len;
-} tdx_attestation_evidence_t;
-
 typedef struct {
 	uint8_t report[8192];
 	uint32_t report_len;
-} snp_attestation_evidence_t;
+} tee_attestation_evidence_t;
 
 typedef struct {
 	char type[ENCLAVE_ATTESTER_TYPE_NAME_SIZE];
 	union {
 		attestation_verification_report_t epid;
-		ecdsa_attestation_evidence_t ecdsa;
-		la_attestation_evidence_t la;
-		tdx_attestation_evidence_t tdx;
-		snp_attestation_evidence_t snp;
+		tee_attestation_evidence_t evidence;
```

Comment:
Please change to 
```C
typedef struct {
	char type[ENCLAVE_ATTESTER_TYPE_NAME_SIZE];
	union {
		attestation_verification_report_t epid;
                struct {
                    uint8_t report[8192];
                    uint32_t report_len;
                };
	};
} attestation_evidence_t;
```

So we could avoid referring report with `evidence->evidence.report`. Now just use `evidence->report`. It looks more clear.

Reply:
Good advice!